Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```
@@ -0,0 +1,413 @@
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterCoat
+  name: winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coat.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coat.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75
+
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterWinterAtmos
+  name: atmos winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.75

```

```suggestion
- type: entity
  parent: ClothingOuterBase
  id: ClothingOuterWinterCoat
  name: winter coat
  description: A heavy jacket made from 'synthetic' animal furs.
  components:
  - type: Sprite
    sprite: Clothing/OuterClothing/WinterCoats/coat.rsi
  - type: Clothing
    sprite: Clothing/OuterClothing/WinterCoats/coat.rsi
  - type: TemperatureProtection
    coefficient: 0.1
  - type: Armor
    modifiers:
      coefficients:
        Slash: 0.95
        Heat: 0.75

- type: entity
  parent: ClothingOuterWinterCoat
  id: ClothingOuterWinterAtmos
  name: atmos winter coat
  description: A heavy jacket made from 'synthetic' animal furs.
  components:
  - type: Sprite
    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
  - type: Clothing
    sprite: Clothing/OuterClothing/WinterCoats/coatatmos.rsi
```

I'm sure you already saw on discord, just adding here for posterity sake, parenting and removing components for all other coats.

---

Everything's been parented now and condensed, thanks for the suggestion